packages/flutter/lib/src/widgets/scrollbar.dart

```diff
@@ -54,7 +55,12 @@ class ScrollbarPainter extends ChangeNotifier implements CustomPainter {
        assert(fadeoutOpacityAnimation != null),
        assert(mainAxisMargin != null),
        assert(crossAxisMargin != null),
-       assert(minLength != null) {
+       assert(minLength != null),
+       assert(padding != null),
+       assert(padding.top >= 0),
```

Comment:
Instead of doing this, add a EdgeInsets.debugAssertIsValid instance method and call that instead.